main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -0,0 +1,94 @@
+# Sell Concert Tickets Smart Contract
+This smart contract is designed to mint and sell event tickets as non-fungible tokens (NFTs) in the form of a semi-fungible asset. In this example there are three categories or classes of tickets:
+- Tickets near the front are the most expensive
+- Tickets in the middle rows are priced between expensive and cheap seats
+- Tickets in the back are the lowest priced
+
+## Contract Setup
+The contract is parameterized with an `inventory` object, which contains information about the different types of tickets available for sale, their prices `tradePrice`, and the maximum number of tickets for each type `maxTickets`. For example:
+```js
+const inventory = {
+  frontRow: {
+    tradePrice: AmountMath.make(istBrand, 3n),
+    maxTickets: 3n,
+  },
+  middleRow: {
+    tradePrice: AmountMath.make(istBrand, 2n),
+    maxTickets: 5n,
+  }
+};
+```
+
+After the contract is initialized, a new ERTP mint for the "Ticket" asset is created:
+```js
+const ticketMint = await zcf.makeZCFMint('Ticket', AssetKind.COPY_BAG);
+const { brand: ticketBrand } = ticketMint.getIssuerRecord();
```

Comment:
this is getting a brand; is the voice-over saying that it's _setting_ the brand?
maybe not critical.